Lykke.Service.LykkeService/Lykke.Service.LykkeService.sln

```diff
@@ -30,6 +30,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "0. Solution items", "0. Sol
 		settings.yaml = settings.yaml
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "5. Host", "5. Host", "{EE7F3159-E06C-497F-96D4-DB2A6B03A6AB}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "6. Contract", "6. Contract", "{14B5AF15-F733-4EF0-9FB9-3E61C4C83ADD}"
```

Comment:
Let's move contract just before client folder in solution since logically it's one level lower than client. - 3. Contract, 4. Client, 5. Tests, 6. Hosts